### PR TITLE
Feature/bin installer

### PR DESCRIPTION
**Pull request by evansde77**
@PerilousApricot @ksnavely

Add a CLI that installs this tool as a binary somewhere specified by the user so it is easier to use withing sublime build packages

### CODE REVIEW

**Comment by PerilousApricot** [comment]
Will this work with a real editor?

## 

It's dark in this basement.

**Comment by evansde77** [comment]
Well, its a shell command, so yeah... (unless you use vi, but in that case you deserve everything you get, and every time you open a file, a puppy dies)

File: src/pyfile_to_module/bin_installer.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+"""
+installer
+
+CLI util to install the pyfile_to_module script as a binary executable
```
Comment (ksnavely) [inline src/pyfile_to_module/bin_installer.py]: It's not really a binary executable

File: src/pyfile_to_module/pyfile_to_module.py [hunk 2]
```diff
@@ -8,9 +8,6 @@
 
 """
 
-__author__ = 'evansde77@gmail.com'
-__version__="0.0.1"
```
Comment (ksnavely) [inline src/pyfile_to_module/pyfile_to_module.py:12]: Is this information tracked elsewhere?

Comment (evansde77) [inline src/pyfile_to_module/pyfile_to_module.py:12]: yeah, its in the **init** module. I need to make cirrus play nicely with adding it to other scripts without upsetting git.

File: src/pyfile_to_module/bin_installer.py (hunk 1 above)
Comment (evansde77) [inline src/pyfile_to_module/bin_installer.py]: updated to say binary like and clarify what it is doing

**Comment by ksnavely** [comment]
LGTM

**Pull request merged**